dpgen/generator/run.py

```diff
@@ -1115,7 +1115,7 @@ def _make_model_devi_native(iter_index, jdata, mdata, conf_systems):
     model_devi_taup = 0.5
     if 'model_devi_taup' in jdata :
         model_devi_taup = jdata['model_devi_taup']
-    mass_map = jdata['mass_map']
+    mass_map = [get_atomic_masses(i) for i in jdata['type_map']]
```

Comment:
This should not override the default config, as one may want to simulate isotopes.

Reply:
updated.

Please check if `jdata['mass_map'] is None` is used correctly